chapter3.adoc

```diff
@@ -134,12 +134,21 @@ after writing `mttp`.
 
 The `msdcfg` is a 32-bit read/write register, formatted as shown in <<MSDCFG>>.
 This CSR is used by M-mode software to specify the active configuration for
-capabilities of the supervisor domain when associated with a hart. The
-extension `Smsdia` uses `msdcfg.SDICN` to specify the active configuration for
-the supervisor domain interrupt controller associated with the hart. Similarly
-the external debug allowed state for a supervisor domain is controlled via the
-`msdcfg.SDEDBALW` bit. Details of `Smsdia` and `Smsdedbg` are described in
-their respective sections in this specification.
+capabilities of the supervisor domain when associated with a hart. +
+The following extensions use the `msdcfg` register to specify additional
+configuration for supervisor domains:
+
+. `Smsdia` uses `msdcfg.SDICN` to specify the active configuration for
+  the supervisor domain interrupt controller associated with the hart.
+. External-debug-allowed state for a supervisor domain is managed via the
```

Comment:
The next two bullets would want to use similar text as first and last bullet in specifying the extension names and field.

Reply:
ack